tests/lpp_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 24 12:12:25 2020
+
+@author: smullally
+"""
+
+
+import exovetter as exo
+import lightkurve as lk
+import astropyfrom astropy import units as u
```

Comment:
```suggestion
from astropy import units as u
```

Reply:
hmm.. I fear this branch is out of date, I know I fixed that.